build/buildall

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -ex
+
+# Install all the versions we support
+mvn -U -Dbuildver=301 clean install -Drat.skip=true -DskipTests
```

Comment:
on my machine scaladoc/javadoc adds 40+ seconds per maven invocation.  Can we add `-Dskip`?

In dist we seem to depend only on classified rapids-4-spark-aggregator. So installing only up to this point should reduce the build time significantly as well if we add  -pl aggregator -am

Reply:
`-Dskip` will skip the scaladoc, but don't we need to specify `-Dmaven.javadoc.skip` to also skip the Javadoc?

Reply:
yes I'll include all suggestions

Reply:
you are right, for the javadoc plugin config element is skip but not the property https://maven.apache.org/plugins/maven-javadoc-plugin/jar-mojo.html#skip . 

Technically we could unify this by adding `<skp>${skip}</skip> ` , or the other way around by adding a more mnemonic flag for scaladoc skip https://davidb.github.io/scala-maven-plugin/doc-jar-mojo.html#skip